targets/bmv2/fuzzer.cpp

```diff
@@ -13,8 +13,8 @@ const Bmv2V1ModelProgramInfo &Bmv2V1ModelFuzzer::getProgramInfo() const {
     return *P4RuntimeFuzzer::getProgramInfo().checkedTo<Bmv2V1ModelProgramInfo>();
 }
 
-InitialP4RuntimeConfig Bmv2V1ModelFuzzer::produceInitialConfig() {
-    p4::v1::WriteRequest request;
+InitialConfig Bmv2V1ModelFuzzer::produceInitialConfig() {
+    auto request = std::make_shared<p4::v1::WriteRequest>();
```

Comment:
Could this be a unique pointer?

Reply:
Changed to unique_ptr. It requires some std::move. https://github.com/fruffy/p4rtsmith/pull/26/commits/2b187d2cf4f07f7460b4fee004396f0f974b2f76